spec/services/proofing/aamva/authentication_client_spec.rb

```diff
@@ -48,71 +48,36 @@
         ).
         and_return(auth_token_request)
       auth_token_request_stub
-
-      allow(Time).to receive(:now).and_return(current_time)
```

Comment:
:oof: what a not great thing to stub, glad we are removing

Reply:
can we remove line 5 now? 

`let(:current_time) { Time.utc(2017) }`

Reply:
yep, thank you, removed.